src/datepicker/datepicker-navigation.ts

```diff
@@ -31,7 +31,7 @@ import {NgbDatepickerI18n} from './datepicker-i18n';
       </div>
       <div class="ngb-dp-arrow" *ngIf="i !== months.length - 1"></div>
     </ng-template>
-    <div class="ngb-dp-arrow right">
+    <div class="ngb-dp-arrow-next right">
```

Comment:
We don't need `.right` here anymore, do we?

Reply:
No, we won't need `.right` here. I'll remove it